client/components/lessons.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux'
+
+const Lessons = (props) => {
+  const { lessons } = props;
+  return (
+    <div>
+      <ul>
+        {lessons.map(lesson => {
+          return (
+            <div key={lesson.id}>
+              <h2>{lesson.name}</h2>
+              <li>Category: {lesson.category}</li>
+              <li>Description: {lesson.description}</li>
+              <li>Image: {lesson.imageUrl}</li>
+              <li>Price: {lesson.price}</li>
+            </div>
+          )
+        })}
```

Comment:
Should the parens and braces be on separate lines to align with the above lines for which they are closing?

Reply:
I think it matches in line with `{lessons.map(lesson => {`. That's why they are all on the same line.

Reply:
I meant that the parens on line 19 is closing the return statement on line 6, and the second brace on line 19 is closing the function opened on line 4

Reply:
Currently they're all in line with the statement opened on line 9

Reply:
Everyone open to using prettify in VS Code moving forward?

Reply:
Okay cool, I didn't see it. Makes sense!